### DISCUSSION TITLE
⏳ 🔗 Support for ACF Repeaters, Flexible Content and Relationships

### DISCUSSION THREAD

**Discussion by c2wd** [elementor/elementor#25394] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

It will solve a lot of problems and the need for custom code and/or third party plugins.

### Describe the solution you'd like.

Support for ACF repeaters, flexible content and relationships in the Loop Builder. This will also allow for dynamic content to be utilized in List, Toggles, Accordions, etc.

### Describe alternatives you've considered.

Bricks Builder. Hate to say it, but Bricks Query Loop natively supports these features out of the box. 
Dynamic.coo
Custom Code

### Additional context

These features are now being incorporated into competitors because Elementor users have been asking for these features numerous times over the years and this has yet to be implemented into Elementor.  As I have been an avid Elementor user for years, I have always found that the no support for these in the Loop Builder is mind boggling as a lot of users that use ACF rely on these types of fields to build more complex websites.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by Digitalfastmind** [comment]
It's incredible that this basic feature is not being developed and added to Ele, requiring another plugin for this is not ideal and also impacts on the performance of the projects based on complexity.

**Comment by Tapulli69** [comment]
Sarebbe l'ora
